templates.yaml

```diff
@@ -4,44 +4,77 @@ ag_news:
       \ or technology? ||| \n{{[\"World politics\", \"Sport\", \"Business\", \"Technology\"\
       ][label] }}"
     name: test1
+    output_fn: ''
+    prompt_fn: ''
     reference: example template_1
   basic: !Template
     jinja: "{{text}} \nIs this text an example of news about world politics, sports,\
       \ business, or technology? ||| \n{{[\"World politics\", \"Sport\", \"Business\"\
       , \"Technology\"][label] }}"
     name: basic
+    output_fn: ''
```

Comment:
Looks like this reverted to an old version of `templates.yaml`. There shouldn't be any *_fn fields anymore.

Reply:
The changes have been updated!